examples/tutorial.ipynb

```diff
@@ -743,22 +739,8 @@
   }
  ],
  "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
   "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "name": "python"
   }
  },
  "nbformat": 4,
```

Comment:
Don't forget to update the cleanvision-examples/ version of this notebook after you've finalized this one and merged this PR